src/scripts.js

```diff
@@ -69,6 +70,10 @@ var glide = new Glide('.glide', {
 
 glide.mount();
 
+let cookbookFlag = false;
+
+let currentRecipeDisplay;
```

Comment:
could we have used `store.user.favoriteRecipeRepo.allrecipes` in place  of `currentRecipeDisplay` to specify when we wanted the display to reload using the users recipes?